.github/ISSUE_TEMPLATE/propose-talk-workshop.md

```diff
@@ -11,16 +11,16 @@ assignees: ''
 
 One can submit a talk on anything related to Python or Open Source and below are the list of categories -
 
-1. Game Design and 3D Modelling - Python in developing games, 3-D modelling and animation
-2. Embedded Python and IOT - MicroPython, Python on Hardware, Robotics, Arduino and Raspberry Pi
-3. Culture and society - Diversity, health, productivity, workspace issues, privacy, community building, coding for causes
-4. Core Python - Language Features, Python Implementations, Extending Python and Standard Library, language internals
-5. Data Science, Machine Learning, and AI
-6. Desktop Applications - Qt, GTK+, Tkinter, Gnome, KDE, Accessibility
-7. Scientific Computing - Python usage in scientific computing and research. GIS, Mathematics, Simulations
-8. Developer tools and automation - Testing, CI/CD, Containers, Orchestration, Logging and Monitoring, Cloud, SRE
-9. Web development - Web, Apis, Microservices
-10. Networking and Security - Network Programming, Network Security and Encryption
+1. Game Design and 3D Modelling - Python in developing games, 3-D modelling and animation-
```

Comment:
@progmatic-99 - these hyphens, in the end, won't make sense. Can you instead add a period(.) If that can help you write the parser.

Reply:
I will test these changes out and share the results with you.

Reply:
I can change those hyphens to period(.)

Reply:
Yeah actually hyphens are used in MD for Bulleted lists so better to check if we can do without it